File: source/Blink.c
#include <stdio.h>
#include <wiringPi.h>

const int LED = 16;

int main (void) {
  if (wiringPiSetupGpio()!=-1) {
    printf("Raspberry Pi LED blinking example\n") ;
    pinMode(LED, OUTPUT);
    for (int loop=0; loop<3; loop++) {
      digitalWrite (LED, HIGH); printf("on\n"); // 3.3 Volt
      delay(500);
      digitalWrite(LED, LOW); printf("off\n"); // 0 Volt
      delay(500);
    }
    pinMode(LED, INPUT);
    return 0;
  }
  printf("wiringPiSetup error"); 
  return 1;
}

File: source/Blink.cs
namespace Raspjamming
{
	using System;
	using System.Threading;

	static class Blink
	{
		const int LED = 16;

		static int Main()
		{
			if (WiringPi.WiringPiSetupGpio() != -1)
			{
				Console.WriteLine("Raspberry Pi LED blinking example");
				WiringPi.PinMode(LED, WiringPi.Output);
				for (int loop = 0; loop < 3; loop++)
				{
					WiringPi.DigitalWrite(LED, WiringPi.High);
					Console.WriteLine("on");
					Thread.Sleep(500);
					WiringPi.DigitalWrite(LED, WiringPi.Low);
					Console.WriteLine("off");
					Thread.Sleep(500);
				}
				return 0;
			}
			Console.WriteLine("wiringPiSetup error");
			return 1;
		}
	}
}
